src/main/java/com/admin/SpringBootDepartmentalStore/bean/Customer.java

```diff
@@ -0,0 +1,39 @@
+package com.admin.SpringBootDepartmentalStore.bean;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import javax.persistence.*;
+import javax.validation.constraints.Email;
+import javax.validation.constraints.NotEmpty;
+
+import edu.umd.cs.findbugs.annotations.SuppressFBWarnings;
+import lombok.Data;
+
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Entity
+@Table(name="Customer")
+@Data
+@SuppressFBWarnings(value = {"EI_EXPOSE_REP", "EI_EXPOSE_REP2"})
+public class Customer {
+	@Id
+	//@GeneratedValue(strategy = GenerationType.AUTO)
```

Comment:
Should be Auto generated, Can use Identity as well. Please read about this as well